sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/AnalyzeOperationResult_internal.cs

```diff
@@ -12,13 +12,31 @@ namespace Azure.AI.FormRecognizer.Models
     /// <summary> Status and result of the queued analyze operation. </summary>
     internal partial class AnalyzeOperationResult_internal
     {
+        /// <summary> Initializes a new instance of AnalyzeOperationResult_internal. </summary>
+        internal AnalyzeOperationResult_internal()
+        {
+        }
+
+        /// <summary> Initializes a new instance of AnalyzeOperationResult_internal. </summary>
+        /// <param name="status"> Operation status. </param>
+        /// <param name="createdDateTime"> Date and time (UTC) when the analyze operation was submitted. </param>
+        /// <param name="lastUpdatedDateTime"> Date and time (UTC) when the status was last updated. </param>
+        /// <param name="analyzeResult"> Results of the analyze operation. </param>
+        internal AnalyzeOperationResult_internal(OperationStatus status, DateTimeOffset createdDateTime, DateTimeOffset lastUpdatedDateTime, AnalyzeResult_internal analyzeResult)
+        {
+            Status = status;
+            CreatedDateTime = createdDateTime;
+            LastUpdatedDateTime = lastUpdatedDateTime;
+            AnalyzeResult = analyzeResult;
+        }
+
         /// <summary> Operation status. </summary>
-        public OperationStatus Status { get; set; }
+        public OperationStatus Status { get; internal set; }
```

Comment:
Do we need internal setters with the constructor that takes all the parameters?

Reply:
We don't use new constructors for deserialization yet.

Reply:
Got it!